neural_compressor/torch/quantization/config.py

```diff
@@ -503,6 +506,76 @@ def get_default_sq_config() -> SmoothQuantConfig:
     return SmoothQuantConfig()
 
 
+######################## HQQ Config ###############################
+@register_config(framework_name=FRAMEWORK_NAME, algo_name=HQQ, priority=PRIORITY_HQQ)
+class HQQConfig(BaseConfig):
+    # Half-Quadratic Quantization (HQQ), more details:
+    # https://mobiusml.github.io/hqq_blog/
+    # Ported from https://github.com/mobiusml/hqq
+
+    name = HQQ
+    params_list = [
+        "nbits",
```

Comment:
Please keep the same name as RTN.

Reply:
Thanks, aligned it in [b6812af](https://github.com/intel/neural-compressor/pull/1597/commits/b6812afaeeeee6f20b5820b14db5804796e54e50).